debian/rules

```diff
@@ -2,12 +2,14 @@
 # -*- makefile -*-
 
 export DH_OPTIONS
+PYTHON_BINARY := $(shell { command -v python3.4 || command -v python3.3 || command -v python2.7; } 2>/dev/null)
```

Comment:
travis runs test for py27, py33, py34 only.
https://github.com/Yelp/docker-custodian/blob/master/.travis.yml

therefore no python3.5 until somebody add it.